src/util/utils.ts

```diff
@@ -1,15 +1,20 @@
 import { ContractKit, newKitFromWeb3 } from '@celo/contractkit'
 import Web3 from 'web3'
-import { WEB3_PROVIDER_URL } from '../config'
 
 let contractKit: ContractKit
-
```

Comment:
nit/question: should we set process env to a const if we're using it a few places e.g. `const web3Provider = process.env.WEB3_PROVIDER_URL`?